app/views/partials/_user.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="user-flex">
   <div class="user-border photo-width">
-    <img class = "image" src= <%= user.photo %> alt = "Jim Carrey">
+    <img class = "image" src= <%= user.photo %> alt = "Photo">
   </div>
```

Comment:
[OPTIONAL]
- In your `views` files/partials, for every `likesCounter, commentsCounter, and postsCounter`, I would suggest you add a conditional rendering like this `Number of posts: <%= user.postsCounter || 0 %>` statement so that they won't just display something like `Number Of Posts: ` but this: `Number of posts: 0` when no post, comment, or like has been added.

- I noticed that some of your files, columns, and routes are not totally in line with the Ruby On Rails Naming convention.
Columns such as `postsCounter` should be written in `snake_case`.
Controller file names should be pluralized and this should be the same for the `routes` as well.

Kindly note that Ruby on Rails favors `convention over configuration`.

You can learn more about Rails Naming Convention [here](https://gist.github.com/iangreenleaf/b206d09c587e8fc6399e)